app/setup_handlers.go

```diff
@@ -5,19 +5,21 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
 )
 
-const releaseVersion = "v8.0.0"
+const releaseVersion = "v9.0.0"
 
 func SetupHandlers(app *App) {
 	app.UpgradeKeeper.SetUpgradeHandler(releaseVersion, func(ctx sdk.Context, plan types.Plan, vm module.VersionMap) (module.VersionMap, error) {
 		app.Logger().Info("Running upgrade handler for " + releaseVersion)
 
-		// Updated version map to thelatest consensus versions from each module
+		// Updated version map to the latest consensus versions from each module
 		for m, mb := range app.mm.Modules {
 			vm[m] = mb.ConsensusVersion()
 		}
-
+		vm[observertypes.ModuleName] = vm[observertypes.ModuleName] - 1
```

Comment:
@kingpinXD you wanna keep these in v9?

Reply:
we have a migration for observer store 
https://github.com/zeta-chain/zeta-node/blob/31d033a3cec3872249056d7215c4428bc8622d64/x/observer/migrations/v2/migrate.go#L11-L28